### ISSUE TITLE
the development server returned response error code 500, Duplicated files or mocks. Please check the console for more info

### ISSUE THREAD

**Issue by lutianxiong** [facebook/react-native#25796]
React Native version:
![QQ截图20190724101659](https://user-images.githubusercontent.com/42662202/61759890-528f5600-adfc-11e9-912d-8d7d0314c102.png)

## Steps To Reproduce

1. react-native init HelloWorld
2. react-native run android

Describe what you expected to happen:
the development server returned response error code 500, Duplicated files or mocks. Please check the console for more info.

Snack, code example, screenshot, or link to a repository:
![QQ截图20190724100947](https://user-images.githubusercontent.com/42662202/61759932-76529c00-adfc-11e9-9575-fa43521aea03.png)
![QQ截图20190724101236](https://user-images.githubusercontent.com/42662202/61759934-76529c00-adfc-11e9-8850-ff48c4ca11b4.png)
![QQ截图20190724101244](https://user-images.githubusercontent.com/42662202/61759935-76529c00-adfc-11e9-96fb-228370d1ff4b.png)

**Comment by boguszjelinski** [comment]
I get exactly the same with my newly (today) installed Android Studio/SDK and ReactNative.

**Comment by VeirryAu** [comment]
having the same error

**Comment by VeirryAu** [comment]
ios: try to clean and rebuild your apps using xcode. (Worked for me)

android: try to clean (./gradlew clean) and rebuild your apps.

**Comment by finntenzor** [comment]
Having the same error and I have no idea how to solve it.

**Comment by finntenzor** [comment]
I removed the file "node_modules/react-native/template/package.json" and it worked.

**Comment by iyegoroff** [comment]
This happens because you named your app "HelloWorld" - it clashes with "name" field in template's package.json file.

**Comment by lutianxiong** [comment]
> This happens because you named your app "HelloWorld" - it clashes with "name" field in template's package.json file.

you are right. thanks!

**Comment by bk1004** [comment]
> > This happens because you named your app "HelloWorld" - it clashes with "name" field in template's package.json file.
> 
> you are right. thanks!

I see you've run "react-native init HelloWorld" so the name field should be "HelloWorld" in package.json file, isn't it? 
How did you solve the problem? did you just remove the package.json file? 

I did "react-native init AwesomeProject" same as the guide, but still "HelloWorld" shown in package.json file. So I changed to "AwesomeProject" in the name field, but still getting the same error message.

**Comment by finntenzor** [comment]
> > > This happens because you named your app "HelloWorld" - it clashes with "name" field in template's package.json file.
> > 
> > 
> > you are right. thanks!
> 
> I see you've run "react-native init HelloWorld" so the name field should be "HelloWorld" in package.json file, isn't it?
> How did you solve the problem? did you just remove the package.json file?
> 
> I did "react-native init AwesomeProject" same as the guide, but still "HelloWorld" shown in package.json file. So I changed to "AwesomeProject" in the name field, but still getting the same error message.

You can try to remove "node_modules/react-native/template" folder. It may work.

**Comment by bk1004** [comment]
> > > > This happens because you named your app "HelloWorld" - it clashes with "name" field in template's package.json file.
> > > 
> > > 
> > > you are right. thanks!
> > 
> > 
> > I see you've run "react-native init HelloWorld" so the name field should be "HelloWorld" in package.json file, isn't it?
> > How did you solve the problem? did you just remove the package.json file?
> > I did "react-native init AwesomeProject" same as the guide, but still "HelloWorld" shown in package.json file. So I changed to "AwesomeProject" in the name field, but still getting the same error message.
> 
> You can try to remove "node_modules/react-native/template" folder. It may work.

Yes, I've tried that but still no luck...

**Comment by lutianxiong** [comment]
> > > > > This happens because you named your app "HelloWorld" - it clashes with "name" field in template's package.json file.
> > > > 
> > > > 
> > > > you are right. thanks!
> > > 
> > > 
> > > I see you've run "react-native init HelloWorld" so the name field should be "HelloWorld" in package.json file, isn't it?
> > > How did you solve the problem? did you just remove the package.json file?
> > > I did "react-native init AwesomeProject" same as the guide, but still "HelloWorld" shown in package.json file. So I changed to "AwesomeProject" in the name field, but still getting the same error message.
> > 
> > 
> > You can try to remove "node_modules/react-native/template" folder. It may work.
> 
> Yes, I've tried that but still no luck...

try using react-native init AwesomeProject in an empty folder.

**Comment by mikevbaker** [comment]
Here's an update for the thread. Having discovered that react-native init doesn't support the current node.js (12.13.0 as of this writing on 11/16/2019). I un-installed Node.JS and installed 12.10.0 and managed to get the files built using `react-native init MyAwesomeProject --template typescript`.
Then I read through these comments and realized that the "name" property in the template folder will be whatever you called your project when you created it. I tried a new folder with a different name and sure enough, the name matched and I got the "Duplicated files or mocks" message.
I cleaned the build and then I changed the name in the template folder's copy of package.json to a different name.
After rebuilding the app started working. I'm not sure if this is creating duplicate instances with different names in the app. I'll try to remember to come back here if this becomes a problem.
